### DISCUSSION TITLE
ykman无法识别canokey

### DISCUSSION THREAD

**Discussion by kagiyamahina2023** [canokeys/canokey-product#23] [Q&A] [answered]
YubiKey Manager版本5.2.1
执行命令ykman -r Canokey piv keys import 9a rsa2048.pem
提示ERROR: Failed to connect to YubiKey
Canokey固件版本2.0.1

**Comment by MetropolisRaccoon** [comment]
> YubiKey Manager版本5.2.1 执行命令ykman -r Canokey piv keys import 9a rsa2048.pem 提示ERROR: Failed to connect to YubiKey Canokey固件版本2.0.1

Hi @kagiyamahina2023, I encountered the same situ. 

Here is my bypass. Just install an earlier version of ykman from its release page https://developers.yubico.com/yubikey-manager-qt/Releases/. ykman 4.0.7 works for me. It is shipped with yubikey-manager-qt-1.2.4 which can be downloaded from https://developers.yubico.com/yubikey-manager-qt/Releases/yubikey-manager-qt-1.2.4.tar.gz. You can experiment with any other versions at your own will.

Good luck!

**Comment by dangfan** [comment, marked as answer]
请使用：https://github.com/canokeys/canokey-manager
